AUTHORS

```diff
@@ -95,10 +101,13 @@ Sergey Zveroboev <szveroboyev@yandex-team.com>
 Stepan Kalinin <mrkastep@gmail.com>
 Timur Iskhakov <me@timur.is>
 Timur Sufiev
+Vadim Skipin
 Vasilii Kalmykov <kvk1920@ytsaurus.tech>
 Vasiliy Morozov <greatdraken@gmail.com>
-Vadim Skipin
 Veronika Ivannikova
+Viktor Rozaev <vrozaev@tracto.ai>
+Vitalii Shevchenko <vitshev@tracto.ai>
+Vladimir Chernitsyn <flunt1k@tracto.ai>
```

Comment:
Do we actually have commits from Ivan Bakurskii, Viktor Rozaev, Vitalii Shevchenko and Vladimir Chernitsyn in this repo?

Reply:
No, but this AUTHORS list is the only one we have, and we already have Maxim Efremoff an other front-end guys there, who do not have commits in the main repo. We agreed on having a single AUTHORS list instead of per-repo long time ago, when we just moved to OSS.

Everybody you mentioned have either commits in k8s operator repo or UI repo.

Reply:
Ok, I agree